inventory/sample.gcp.example.com.d/inventory/hosts

```diff
@@ -0,0 +1,38 @@
+localhost connection=local ansible_become=false
+
+[cluster_hosts:children]
+OSEv3
+
+[OSEv3:children]
+masters
+nodes
+etcd
+
+[masters:children]
+tag_master-env1
```

Comment:
I suspect that these `-env1` portions of the groups are tied to the `env_id` - can we please ensure to capture that these need to be updated when the `env_id` is changed?

Reply:
@meeting

Reply:
done